nlp/front/service/src/main/kotlin/ParserService.kt

```diff
@@ -282,10 +292,11 @@ object ParserService : Parser {
                                 core.supportValuesMerge(it.entityType)
                             )
                         },
-                        intentProbability,
+                        realIntentProbability,
```

Comment:
Does it mean that before those changes we had unknown with probabilities different from 1 ?

Reply:
yes. we send back that the NLP found 'unkonw' at 40%, except that this is not true, the 40% is for the intent actually detected.